docs/registry/entities/aws-ecs.md

```diff
@@ -0,0 +1,35 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_entry.md.j2 -->
+
+
+
+# Aws Ecs
```

Comment:
leaving comment here, but it's a general question:

what would we do for metrics/events/spans in the future - would we group them by the root namespace or render each in a  separate file? 

I think it makes sense to group them (e.g. all HTTP metrics in one doc) -  having one per file feels too granular. But that's what we generate here and it matches current resource docs layout

Reply:
Yeah - I think having one per file DOES make a lot of sense. I can update the entity registry to do that too.